bookshelter-app/src/components/BookCategory/index.tsx

```diff
@@ -4,16 +4,16 @@ interface BookCategoryProps {
   name: string;
   totalBooks: number;
   color: string;
-  onClick: () => void;
   categorySlug: string;
+  onClick: () => void;
 }
 
 const BookCategory = ({
   name,
   totalBooks,
   color,
-  onClick,
   categorySlug,
+  onClick,
```

Comment:
remove it if not use

Reply:
when user click BookCategory in Mobile app and close CategoryList